file-downloader/Cargo.toml

```
@@ -0,0 +1,19 @@
+[package]
+name = "solana-file-downloader"
+description = "Solana File Download Utility"
+documentation = "https://docs.rs/solana-file-downloader"
```

nit: I lean towards calling this `solana-file-download` since it's really just exposing the function
```suggestion
name = "solana-file-download"
description = "Solana File Download Utility"
documentation = "https://docs.rs/solana-file-download"
```